src/ui/index.ts

```diff
@@ -23,6 +23,3 @@ export * from './TextArea/TextArea';
 export * from './TextField/TextField';
 export * from './Tooltip/Tooltip';
 export * from './Icon/Icon';
-
-// icons
-export * from './icons';
```

Comment:
whats the deal here? just curious

Reply:
I was thinking that it was no longer needed.  Perhaps I'm wrong?

Reply:
.... ah, that was a relic of when there were some icons stored in /ui.  yea, no longer need to import those.  
<img width="307" alt="Screen Shot 2022-09-16 at 4 20 50 PM" src="https://user-images.githubusercontent.com/100873710/190829835-56e702e3-79b2-41a4-83b8-b37273b7c64b.png">